### PR TITLE
rtv 1.8.0 (new formula)

### PR DESCRIPTION
**Pull request by brettmichael**
rtv 1.8.0

A command line reddit client located here: https://github.com/michael-lazar/rtv

Requested here: https://github.com/Homebrew/homebrew/issues/44129

My first formula but all guidance, brew audit, and brew test all pass without warning or error.

### CODE REVIEW

File: Library/Formula/rtv.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  sha256 "90828ad016d669dfc162ff958124700abcce7f2cf1aedcf5be90fe97f4552764"
+
+  depends_on :python3
+
+  resource "decorator" do
+    url "https://pypi.python.org/packages/source/d/decorator/decorator-4.0.6.tar.gz"
+    sha256 "1c6254597777fd003da2e8fb503c3dbf3d9e8f8d55f054709c0e65be3467209c"
+  end
+
+  resource "update_checker" do
```
Comment (DomT4) [inline Library/Formula/rtv.rb]: What's this being used for?

Comment (brettmichael) [inline Library/Formula/rtv.rb]: It's a dependency of PRAW which is the python Reddit library rtv uses - sounded like it was used to check pypi packages are up to date so doubt it's doing much/anything at runtime.

https://github.com/praw-dev/praw/blob/master/setup.py

Comment (DomT4) [inline Library/Formula/rtv.rb]: Alright, great. As long as the tool isn't trying to auto-update itself that's fine.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/f5a1bd52d764911be146fd8e1a0d96974b9c8f47. Thank you for your contribution to Homebrew @brettmichael; we appreciate it! :smiley_cat:

**Pull request closed without merging**